forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -548,6 +548,7 @@ else if (Iterables.any((CardCollectionView) params.get(AbilityKey.LastStateGrave
         // do ETB counters after zone add
         if (!suppress) {
             if (toBattlefield) {
+                game.getTriggerHandler().registerActiveTrigger(copied, false);
```

Comment:
The same call already happens in L592

do we need this twice now or can we just move the `table.replaceCounterEffect` after?

Reply:
I didn't fully debug it, but it probably needs to be happen before static abilities